src/pages/Mypage.css

```
@@ -47,6 +47,49 @@
+    gap: 6px;
+    padding: 0;
+    background: transparent;
+  }
+
+  .sidebar ion-item {
+    flex: 1;
+    --min-height: 42px;
+    --padding-start: 0;
+    --inner-padding-end: 0;
+    --inner-border-width: 0;
+    text-align: center;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)